Docs/Configuration.tex

```diff
@@ -382,14 +382,14 @@ \subsection{Configuration Structure}\label{configuration-structure}
   \hyperref[uefi]{\texttt{UEFI}}
 \end{itemize}
 
-It is possible to perform basic validation of the configuration by using
-\texttt{ocvalidate} utility. Please note, that \texttt{ocvalidate}
+Basic validation of the \texttt{OC\ config} file is possible using
+the \texttt{ocvalidate} utility. Please note that \texttt{ocvalidate}
 must match the used OpenCore release and may not be able to detect all
-configuration flaws present in the file.
+configuration issues present in the file.
 
-\emph{Note}: Currently most properties try to have defined values even if not
-specified in the configuration for safety reasons. This behaviour should not
-be relied upon, and all fields must be properly specified in the configuration.
+\emph{Note}: For system integrity, most properties typically have predefined values
```

Comment:
Remove "most" (since with "most ... typically" suggest one or other should be removed, or it should be reworded)

Reply:
Not sure. "Most" is attached to "properties" Most of the properties have this.
Anyway, will review

Reply:
I suggest no, either "most properties have", or "(all, implicitly) properties try to have", if you see where I'm coming from?

Reply:
Will review. Thanks